src/components/annotations/ImageAnnotation.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from "react";
```

Comment:
Move all the annotation files into 'annotations' folder.
I didn't change any code.